Packs/CiscoWebExFeed/Integrations/CiscoWebExFeed/CiscoWebExFeed.yml

```diff
@@ -82,34 +86,34 @@ configuration:
   name: insecure
   required: false
   type: 8
+  section: Connect
+  advanced: true
 - display: Use system proxy settings
   name: proxy
   required: false
   type: 8
-description: The WebEx IP Address and Domain web site provided by Cisco to document
-  IPs and Domains used by WebEx. The WebEx Feed integration fetches indicators from
-  the web page, with which you can create a list (allow list, block list, EDL, etc.)
-  for your SIEM or firewall service to ingest and apply to its policy rules.
+  section: Connect
+  advanced: true
+description: Use the Cisco WebEx Feed integration to fetch indicators from WeBex.
 display: Cisco WebEx Feed
 name: Cisco WebEx Feed
 script:
   commands:
   - arguments:
-    - defaultValue: "100"
-      description: The maximum number of results to return. The default value is 10.
+    - defaultValue: "20"
+      description: The maximum number of results to return.
       name: limit
     - auto: PREDEFINED
       defaultValue: Both
-      description: The indicator type. Can be "IPs", "URLs", or "Both". The default
-        value is "Both".
+      description: The indicator type. Can be "CIDR", "DOMAIN", or "Both".
```

Comment:
no reason to list the possible values.  When generating the README, the values automatically appear. 
So this should be: The indicator type.